WPAndroid Integration.md

```diff
@@ -0,0 +1,113 @@
+
+#### How it works:
+
+When [settings.gradle](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/settings.gradle) finds `localGutenbergMobilePath` in `local-builds.gradle`, it'll substitute the below binary dependency with the local folder:
+
+```
+implementation "$rootProject.gradle.ext.gutenbergMobileBinaryPath:$rootProject.ext.gutenbergMobileVersion"
+```
+
+`react-native-bridge` is setup to include all the other projects, so no other action is necessary.
+
+---
+
+### Deploy a new version of `react-native-bridge` from CI
+
+There are a few different ways to do this:
+
+1. Open a new PR or push a new commit to an open PR which will be published as `<PR number>-<commit full SHA1>`
+2. Merge a PR to `develop` which will be published as `<develop>-<commit full SHA1>`
+3. Create a new tag which will be published as `<tag name>`
+
+**How it works:**
+
+CI will run the following commands:
+
+- `npm ci`
+- `npm run bundle:android`
+- `gutenberg/packages/react-native-bridge/android/publish-aztec-and-bridge.sh <version>`
+
+---
+
+### Manually deploy a new version of `react-native-bridge`
+
+_**IMPORTANT:** This should only be used for testing.
+We shouldn't commit a manually deployed version to WPAndroid's `develop` because the process bypasses the guarantees CI would otherwise give us._
+
+- Make sure you run `npm install` or `npm ci` first
+- Run `npm run bundle:android` so the JS bundle is created
+- Run `publish-aztec-and-bridge.sh` script in `react-native-bridge`, giving it a version name such as `locally-built-by-<your_name>-<commit full SHA1>`
+
+**How it works:**
+
+Besides checking a few things, `publish-aztec-and-bridge.sh` script will clean the project, copy the bundled JS file into its assets, publish `react-native-aztec` and use that version to build and publish `react-native-bridge`.
+
+---
+
+### Update WPAndroid's binary version
+
+- Find [the version deployed from CI](#deploy-a-new-version-of-react-native-bridge-from-ci) or [the version you've manually deployed](#manually-deploy-a-new-version-of-react-native-bridge)
+- Update `ext.gutenbergMobileVersion` property in [build.gradle](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/build.gradle) to the new version
+
+In order to test this, make sure `localGutenbergMobilePath` in your `local-builds.gradle` file is commented out as otherwise the binary version will be ignored.
+
+---
+
+## How to
+
+### How to test a gutenberg-mobile PR in WPAndroid
+
+Assuming that there is no open WPAndroid PR yet:
+
+1. Setup WPAndroid to [work with local gutenberg-mobile checkout](#work-with-local-gutenberg-mobile-checkout)
+2. Switch `gutenberg-mobile` to the branch you're testing
+3. Run the app
+
+---
+
+### How to share a WIP WPAndroid APK
+
+1. Open a `gutenberg-mobile` PR which will [publish a new version](#deploy-a-new-version-of-react-native-bridge-from-ci) or [manually publish a new version](#manually-deploy-a-new-version-of-react-native-bridge)
```

Comment:
I would rephrase this a bit to something like "Either (a) open a `gutenbeg-mobile` PR which will publish a new version, or (b) manually publish a new version". That way there's no chance of someone reading this as saying that opening the PR will "publish a new version or manually publish a new version".

Reply:
Addressed in 6a5ca7effc502ea798275d8ced92a54755d14942.